portal/guides/on-demand-minting.mdx

```diff
@@ -1,13 +1,15 @@
 ---
-title: "Using signature based minting"
-author: "Raza Zaidi"
+title: "Minting NFTs on demand using signatures"
+author: "Raza Zaidi and Ayush Pathak"
 authorAvatar: "/assets/portal/authors/raza-zaidi.jpg"
+authorAvatar2: "/assets/portal/authors/ayush.jpg"
 authorTwitter: "https://twitter.com/razacodes"
-draft: true
+authorTwitter2: "https://twitter.com/ayshptk"
+draft: false
 tags: [nft-drop, signature, mint]
 summary: "This guide shows how to create a signature and allow anyone holding that signature to mint an NFT!"
-image: "/assets/portal/guides/minting-with-signature/thumbnail.png"
-date: "2022-01-18"
+image: "/assets/portal/guides/on-demand-minting/thumbnail.png"
```

Comment:
i see there's a video `demo.mov` in this PR, but it's not showing up anywhere in the guide itself?

Reply:
need to update this. will record a new screen session later